pkg/codegen/python/gen_test.go

```diff
@@ -56,6 +56,8 @@ func TestGeneratePackage(t *testing.T) {
 						assert.Contains(t, string(file), "class Resource(pulumi.CustomResource):")
 						// Remote option not set
 						assert.NotContains(t, string(file), "remote=True)")
+						// CustomResource class contains a get method
+						assert.Contains(t, string(file), "def get(resource_name: str,")
```

Comment:
I'd love it if we could make these tests check against either files on disk or literal strings rather than looking for substrings.

Reply:
I considered that approach, but I wasn't sure if those tests would be unnecessarily brittle since any codegen change would require updating the file. I don't have a strong opinion on it, so happy to update if you do.

Reply:
Done